examples/js/exporters/GLTFExporter.js

```diff
@@ -675,6 +681,11 @@ THREE.GLTFExporter.prototype = {
 				gltfMaterial.pbrMetallicRoughness.metallicFactor = material.metalness;
 				gltfMaterial.pbrMetallicRoughness.roughnessFactor = material.roughness;
 
+			} else if ( material instanceof THREE.MeshBasicMaterial ) {
+
+				gltfMaterial.pbrMetallicRoughness.metallicFactor = 0.0;
+				gltfMaterial.pbrMetallicRoughness.roughnessFactor = 0.9;
```

Comment:
Where are these numbers 0.0 and 0.9 from?

Reply:
Ah, from here https://github.com/KhronosGroup/glTF/pull/1163/files#diff-a510f3afb3af7966176e2c4d4f200efbR105